specs/opcode/02MUL.md

```diff
@@ -0,0 +1,96 @@
+# MUL op code
```

Comment:
I'm having some difficulty reading the latex equations and tables.  It seems that there are extra double `\` everywhere, where only one is required for latex.  For example, if I copy the document into hackmd this is what I see https://hackmd.io/b9qqwnwnTpCxJlrsv1D58w?both
What tool did you use to render this markdown?

Reply:
I agree with u, I think it's our mistake.

Reply:
@noel2004

Reply:
I will check it. I have simply copied the exported md file from hackmd before.

Reply:
It is the lint / fmt procedure `mdformat --number` adding extra slash into the embedded latex code. It seems we can not use latex syntax inside the documents or it can not pass the lint checking. So I have to provide linking of the references only.

Reply:
Finally I turn the original reference into a plain markdown format with unicode symbols.